arch/c4_context.md

```diff
@@ -0,0 +1,49 @@
+# Level 1: Radix System Context
+
+This page implement the first "C" in the [C4Model](https://c4model.com/#coreDiagrams): the (System) **C**ontext.
+
+All production Systems (External and Radix) and Users that are commonly found on Radix networks are introduced on this page.
+
+![uncached image](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
```

Comment:
Image seems to imply that can only gossip to to boot nodes across shards.

Reply:
good catch, I just drew random arrows, not thinking about this at all.